lib/elastic_mappings.rb

```diff
@@ -13,8 +13,7 @@ module ElasticMappings
       publish_start_on: { type: 'date', format: 'YYYY-MM-dd' },
       publish_end_on: { type: 'date', format: 'YYYY-MM-dd', null_value: '9999-12-31' },
       title: { type: 'text', term_vector: 'with_positions_offsets' },
-      match_keyword_values_only: { type: 'boolean',
-                                   index: :not_analyzed,
+      match_keyword_values_only: { type: 'keyword',
```

Comment:
Why the change from `boolean` to `keyword`? `match_keyword_values_only` should only include boolean values.